src/AutoRest.CSharp/Mgmt/Generation/MgmtClientBaseWriter.cs

```diff
@@ -126,9 +136,9 @@ protected internal string AwaitKeyword(bool async)
             return async ? "await" : string.Empty;
         }
 
-        protected internal string OverrideKeyword(bool isInheritedMethod)
+        protected internal string OverrideKeyword(bool isInheritedMethod, bool isVirtual = false)
         {
-            return isInheritedMethod ? "override" : string.Empty;
+            return isInheritedMethod ? "override" : (isVirtual ? "virtual" : string.Empty);
```

Comment:
shouldn't we call the method on line 124?

Reply:
Ah, I forgot we have that function. will change